### ISSUE TITLE
TagKeyword-criterion doesn't work with solr if the query-clause contains multiple words.

### ISSUE THREAD

**Issue by whitefire** [netgen/TagsBundle#37]

**Comment by emodric** [comment]
Thanks, I'll look into it!

For reference, issue seems to arise from (obviously wrong) handling of `Operator::LIKE` in `TagKeyword` visitor: https://github.com/netgen/TagsBundle/blob/master/Core/Search/Solr/Query/Content/CriterionVisitor/Tags/TagKeyword.php#L51-L53

**Comment by emodric** [comment]
Fixed in 0fc3360630abf56e1610a9916377b7027de85fd0, however, only prefix search is possible. The following criteria:

```
$criteria = array(
    new TagKeyword(Criterion\Operator::LIKE, "web de", 'tags'),
);
```

will return all content which is tagged with a tag which keywords starts with "web de".
